sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/SdkMeterProviderBuilder.java

```diff
@@ -29,7 +29,7 @@
   private Resource resource = Resource.getDefault();
   private final ViewRegistryBuilder viewRegistryBuilder = ViewRegistry.builder();
   private final List<MetricReaderFactory> metricReaders = new ArrayList<>();
-  // Default the sampling strategy.
+  // Set the default exemplar filter strategy.
   private ExemplarFilter exemplarFilter = ExemplarFilter.sampleWithTraces();
```

Comment:
Perhaps we should document these defaults (sampleWithTraces and min coll interval) in the class JavaDoc? Looks to be a better place than field doc.